Libraries/src/Amazon.Lambda.ApplicationLoadBalancerEvents/Amazon.Lambda.ApplicationLoadBalancerEvents.csproj

```diff
@@ -3,13 +3,21 @@
     <Import Project="..\..\..\buildtools\common.props" />
 
     <PropertyGroup>
-        <TargetFramework>netstandard2.0</TargetFramework>
+        <TargetFrameworks>netstandard2.0;netcoreapp3.1</TargetFrameworks>
         <Description>Amazon Lambda .NET Core support - Application Load Balancer package.</Description>
         <AssemblyTitle>Amazon.Lambda.ApplicationLoadBalancerEvents</AssemblyTitle>
-        <VersionPrefix>1.0.0</VersionPrefix>
+        <VersionPrefix>2.0.0</VersionPrefix>
         <AssemblyName>Amazon.Lambda.ApplicationLoadBalancerEvents</AssemblyName>
         <PackageId>Amazon.Lambda.ApplicationLoadBalancerEvents</PackageId>
         <PackageTags>AWS;Amazon;Lambda</PackageTags>
     </PropertyGroup>
+
+    <PropertyGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
+        <DefineConstants>NETSTANDARD_2_0</DefineConstants>
```

Comment:
Why not use the built-in `NETSTANDARD_20` and `NETCOREAPP_31` hash-defines?

Reply:
Thanks, I didn't realize there were built in symbols. Updated PR to use them.